### PR TITLE
Removed rating versions

### LINKED ISSUE #187: Better versions for ratings
Currently, `Rating.version()` always return the same value even if the rating has been updated. Therefore, it's not possible to distinguish updates in the rating definition. The `version()` method needs to be updated to return incremental versions when the rating gets updated.

### PR DESCRIPTION
**Pull request by artem-smotrakov**
This closes #187

### CODE REVIEW

File: src/main/java/com/sap/sgs/phosphor/fosstars/model/rating/example/SecurityRatingExample.java [hunk 1]
```diff
@@ -44,8 +42,8 @@ public void makeImmutable() {
    * Initializes a security rating with SecurityScoreExample.
    */
   @JsonCreator
```
Comment (sourabhsparkala) [inline src/main/java/com/sap/sgs/phosphor/fosstars/model/rating/example/SecurityRatingExample.java:44]: Is this still required, if there is no `@JsonProperty`?

File: src/main/java/com/sap/sgs/phosphor/fosstars/model/rating/oss/OssSecurityRating.java [hunk 2]
```diff
@@ -23,10 +22,9 @@
    */
   @JsonCreator
   public OssSecurityRating(
-      @JsonProperty("score") OssSecurityScore score,
-      @JsonProperty("version") Version version) {
+      @JsonProperty("score") OssSecurityScore score) {
 
-    super("Security rating for open-source projects", score, version);
+    super("Security rating for open-source projects", score);
```
Comment (sourabhsparkala) [inline src/main/java/com/sap/sgs/phosphor/fosstars/model/rating/oss/OssSecurityRating.java:27]: Is this score being null checked?

Comment (artem-smotrakov) [reply src/main/java/com/sap/sgs/phosphor/fosstars/model/rating/oss/OssSecurityRating.java:27]: Yeah. Please see `AbstractRating`.

File: src/main/java/com/sap/sgs/phosphor/fosstars/model/rating/example/SecurityRatingExample.java (hunk 1 above)
Comment (artem-smotrakov) [reply src/main/java/com/sap/sgs/phosphor/fosstars/model/rating/example/SecurityRatingExample.java]: Right. I've removed it.

**Pull request merged**